pkg/rsstorage/internal/integration_test/integration_test.go

```diff
@@ -126,20 +139,25 @@ func (s *StorageIntegrationSuite) TearDownTest(c *check.C) {
 // In the tests, we'll typically create two server sets, one set as a source
 // set and another set as a destination set.
 func (s *StorageIntegrationSuite) NewServerSet(c *check.C, class, prefix string) map[string]rsstorage.StorageServer {
-	s3Svc, err := s3server.NewS3Wrapper(&rsstorage.ConfigS3{
-		Bucket:             class,
-		Endpoint:           "http://minio:9000",
-		Prefix:             prefix,
-		EnableSharedConfig: true,
-		DisableSSL:         true,
-		S3ForcePathStyle:   true,
-	}, "")
+
+	endpointURL, _ := url.Parse("http://minio:9000")
+
+	s3Svc, err := s3server.NewS3Wrapper(
+		s3.Options{
+			Region:             "us-east-1",
+			EndpointOptions:    s3.EndpointResolverOptions{DisableHTTPS: true},
+			EndpointResolverV2: &Resolver{URL: endpointURL},
+			UsePathStyle:       true,
+			Credentials:        credentials.NewStaticCredentialsProvider("minio", "miniokey", ""),
+			//ClientLogMode:      aws.LogRequestWithBody | aws.LogResponseWithBody,
+			//RetryMaxAttempts:   1,
+			//Logger: logging.NewStandardLogger(os.Stdout),
```

Comment:
debug code?

Reply:
@nodivbyzero, ah I added those for debugging purposes but decided to leave them in for future use. I added a comment in ea7e1bb to explain that they should only be enabled for debugging since they are a lot of noise to the tests' output.